.github/workflows/lint-go.yaml

```diff
@@ -36,6 +36,22 @@ jobs:
           go mod vendor
           test -z "$(git status --porcelain)" || (echo "please run 'go mod tidy && go mod vendor', and submit your changes"; exit 1)
 
+  license-check:
+    name: Check third party dependencies licenses
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          # renovate: datasource=golang-version depName=go
+          go-version: 1.22.0
```

Comment:
Can't we source it from `go.mod`?

Reply:
I'm following the way it's done for the rest of the CI workflows. I believe we cannot because we want to keep the Go version in the go.mod at 1.22.0 (ie without incrementing the patch version) so keep the same behavior as with older Go version where only major.minor was specified in the go.mod.